mongodb/src/test/java/org/apache/calcite/adapter/mongodb/MongoAdapterTest.java

```diff
@@ -605,12 +607,7 @@ private CalciteAssert.AssertThat assertModel(URL url) {
                     + "    ]\n"
                     + "  }\n"
                     + "}",
-                "{$project: {CITY: '$city', STATE: '$state'}}",
-                "{$group: {_id: {CITY: '$CITY', STATE: '$STATE'}}}",
-                "{$project: {_id: 0, CITY: '$_id.CITY', STATE: '$_id.STATE'}}",
-                "{$group: {_id: '$STATE', CDC: {$sum: {$cond: [ {$eq: ['CITY', null]}, 0, 1]}}}}",
-                "{$project: {STATE: '$_id', CDC: '$CDC'}}",
-                "{$sort: {STATE: 1}}"));
+                "{$project: {STATE: '$state', CITY: '$city'}}"));
```

Comment:
I'm not very familiar with the mongo adapter. However, judging from the plan, the script here seems to have problems. The previous test cases looked correct from the perspective of SQL semantics. But now this plan has lost the `group by` and `order by` clauses. I guess we can check whether the related operations have been completed through Enumerable by adding explainContains.

Reply:
I found the implementation of group and count distinct is not implemented by mongo statements, but through code and API
https://github.com/apache/calcite/blob/618e601b136e92db933523f77dd7af3c1dfe2779/mongodb/src/main/java/org/apache/calcite/adapter/mongodb/MongoToEnumerableConverter.java#L113-L116

https://github.com/apache/calcite/blob/618e601b136e92db933523f77dd7af3c1dfe2779/mongodb/src/main/java/org/apache/calcite/adapter/mongodb/MongoTable.java#L131-L153

Reply:
@NobiGo Thank you for your suggestion, I have added the explainContains, and the plan indicates  the related operations have been completed.